bobtemplates/plone_addon/buildout.cfg.bob

```diff
@@ -1,11 +1,14 @@
 [buildout]
-extends = http://dist.plone.org/release/{{{ plone.version }}}/versions.cfg
+extends =
+    http://dist.plone.org/release/{{{ plone.version }}}/versions.cfg
+    https://raw.githubusercontent.com/plone/plone.app.robotframework/master/versions.cfg
```

Comment:
@hvelarde I don't think this is a good idea. This means buildouts generated by bobtemplates.plone can break at any time if something changes in p.a.robotframework repo. It also means that our travis build can fail at any time without us noticing.